client/README.md

```diff
@@ -0,0 +1,77 @@
+# SDK Client
+
+## Instruction
+
+### Remove remaining data(if exists)
+
+```bash
+> rm -r ~/.gaiad
+> rm -r ~/.gaiacli
+```
+
+### Installation
+
+```bash
+> go get -u github.com/cosmos/cosmos-sdk
+package github.com/cosmos/cosmos-sdk: no Go files in /home/mossid/go/src/github.com/cosmos/cosmos-sdk
+> cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+> git checkout develop
+Branch develop set up to track remote branch develop from origin.
+Switched to a new branch 'develop'
+> dep ensure
+> make install
+go install -ldflags "-X github.com/cosmos/cosmos-sdk/version.GitCommit=be7ec5b" ./cmd/gaia/cmd/gaiad
+go install -ldflags "-X github.com/cosmos/cosmos-sdk/version.GitCommit=be7ec5b" ./cmd/gaia/cmd/gaiacli
+```
+
+### Run node and server
+
+```bash
+# If your are going to connect to an external node(e.g. testnet), skip gaiad init and gaiad start
+> gaiad init --name myname
+{
+  "chain_id": "test-chain-vFxi71",
+  "node_id": "ef0dcdf3b4c90b9f5a1f09975730ae7708861ed6",
+  "app_message": null
+}
+> gaiad start
+...
+# If your are going to connect to an external node, speficy node address with --node flag
+> gaiacli advanced rest-server
+I[06-05|04:26:30.739] Starting RPC HTTP server on tcp://localhost:1317 module=rest-server 
+I[06-05|04:26:30.739] REST server started                          module=rest-server
+```
+
+### Test getting account
+
+```bash
+> cat $HOME/.gaiad/config/genesis.json
```

Comment:
This is unclear. Can you clarify the different steps, and comment? Maybe use the GET method `/balance` from https://github.com/cosmos/cosmos-sdk/pull/1092 as an example

Reply:
`Test getting account` section is for testing the lcd is working correctly. To do this, we first get balance from genesis file(as the "expected' value) and compare it with lcd query result(as the "actual" value). Interchain standard describes the lcd endpoints, which could be wrong if the lcd is malfunctioning, so we cannot use it as the expected value I think?